stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/MessageItem.kt

```diff
@@ -266,7 +276,9 @@ internal fun DefaultMessageItemHeaderContent(messageItem: MessageItemState) {
             }
             ?.let { options ->
                 MessageReactions(
-                    modifier = Modifier.padding(horizontal = 4.dp, vertical = 2.dp),
+                    modifier = Modifier
+                        .clickable(onClick = { onReactionsClick(message) })
```

Comment:
We should put in

`indication = rememberRipple()` here and all that - currently the click is an ugly Compose default black box and this would make it much nicer!

Reply:
Nice catch!